app/views/broker_agencies/profiles/_index.html.erb

```diff
@@ -30,13 +30,13 @@
                 <td><%= broker_agency_profile.entity_kind.titleize %></td>
                 <td><%= broker_agency_profile.market_kind.titleize %></td>
                 <td>
-                <%= link_to 'Send Secure Message'.html_safe,
+                <%= link_to ("#{l10n('send_secure_message')}").html_safe,
```

Comment:
Instead of doing `link_to ("#{l10n('send_secure_message')}").html_safe,` you can use Rails' conventions and make this line easier to read by appending _html to the end of your localization key; e.g. `link_to l10n('send_secure_message_html')`